Fix "error: #pragma GCC diagnostic not allowed inside functions" with GCC < 4.4
https://bugzilla.gnome.org/show_bug.cgi?id=762562
--- glib/gdate.c.orig	2016-02-16 05:10:31.000000000 -0600
+++ glib/gdate.c	2016-02-23 13:43:39.000000000 -0600
@@ -2494,10 +2494,7 @@
        * recognize whether strftime actually failed or just returned "".
        */
       tmpbuf[0] = '\1';
-      #pragma GCC diagnostic push
-      #pragma GCC diagnostic ignored "-Wformat-nonliteral"
       tmplen = strftime (tmpbuf, tmpbufsize, locale_format, &tm);
-      #pragma GCC diagnostic pop
 
       if (tmplen == 0 && tmpbuf[0] != '\0')
         {
